src/assets/data/schedule-data.js

```diff
@@ -71,13 +71,9 @@ let scheduleData = [
         name: "HackNights Time",
```

Comment:
Sunday Schedule:
9:30 AM - Check-in
10:30 AM - Project Expo
12:30 PM - Closing Ceremony
HackNights and ZotHacks Ends can be removed